backend/src/packages/auth/auth.app-plugin.ts

```diff
@@ -11,7 +17,7 @@ import { type AuthPluginOptions } from './libs/types/auth-plugin-options.type.js
 import { jwtPayloadSchema } from './libs/validation-schemas/validation-schemas.js';
 
 const authPlugin = fp<AuthPluginOptions>((fastify, options, done) => {
-  const { userService, jwtService } = options;
+  const { userService, jwtService, stripeRepository } = options;
```

Comment:
Service, service, repository 🥲 . Let's be consistent and respect application layers. Plugins should use services.